packages/waku/src/router/client.ts

```diff
@@ -69,19 +77,23 @@ export function useLocation() {
   return value.loc;
 }
 
+export type LinkProps = PropsWithChildren<
+  {
+    href: string;
```

Comment:
Should we change this to `to`? v0.19.0 would be a good timing for such a breaking change.